lib/pbench/server/api/resources/upload_api.py

```diff
@@ -378,6 +393,21 @@ def put(self, filename: str):
                 status = e.status
             else:
                 self.logger.exception("Unexpected exception in outer try")
+
+            # FIXME:
```

Comment:
Keep the `FIXME`, or keep the current behavior? Or both? 😆 

Since you seemed to agree on the idea that we don't try to audit header validation failures before we create the resource, I think bullet 2 should just go away.

I'd like to capture a `reason` for the failure, and it occurs to me that it's not as hard as I'd thought at the time. We have the HTTP status: so at the level of granularity I've defined it's either `INTERNAL` of the status is a 500 or `CONSISTENCY`. For upload the only "PERMISSION" error would be if there's no `authorization: Bearer <x>` header, and in that case we'll fail long before we bother to audit.

So I'm just going to fix that and remove the first bullet.